src/main/java/net/minestom/server/event/server/StatusRequestEvent.java

```diff
@@ -0,0 +1,61 @@
+package net.minestom.server.event.server;
+
+import net.minestom.server.event.CancellableEvent;
+import net.minestom.server.event.Event;
+import net.minestom.server.network.player.PlayerConnection;
+import net.minestom.server.ping.ResponseData;
+import org.jetbrains.annotations.NotNull;
+
+
+/**
+ * Called by a StatusRequestPacket.
+ * Can be used to modify the {@link ResponseData} response data.
+ */
+public class StatusRequestEvent extends Event implements CancellableEvent {
+
+    private final ResponseData responseData;
```

Comment:
Is there any reason to not just add the response data fields here? Otherwise every use case is just
```java
ResponseData data = event.getResponseData();
// ...
```